recipes/feedgenerator/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.7" %}
+
+package:
+    name: feedgenerator
+    version: {{ version }}
+
+source:
+    fn: feedgenerator-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/f/feedgenerator/feedgenerator-{{ version }}.tar.gz
+    md5: 92978492871342ad64e8ae0ccfcf200c
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - pytz >=0a
```

Comment:
Oops. Thanks for spotting that. (No idea if that was a conda-skeleton bug or I had a mouse/keyboard glitch and edited it without noticing.)